src/tir/schedule/analysis/analysis.cc

```diff
@@ -207,14 +220,14 @@ int CheckCompleteBlockErrorCode(const ScheduleState& self, const StmtSRef& block
 
 static const char* kCompleteBlockDefinition = R"(Definition of a complete block:
 1) All block vars are data parallel
-2) Dominant: the block is the only writer of its output, dominating the reader of its output buffers
+2) Dominant: the block is the only writer of its output, dominating the reader of its output buffers under the given scope.
```

Comment:
for `kCompleteBlockDefinition`, I still prefer to keep it as `under the block scope`